<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::PageLogin</class>
 <widget class="QWidget" name="ns::PageLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YesPo Service Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widgetTop {
	background: transparent;
}

#widgetTop #labelCaption {
	font-size: 28px;
	font-weight: bold;
	color: #ffffff;
}

#widgetTop #pushButtonBack {
	border-image: url(:/images/ui/login_back.png);
}

#widgetTop #pushButtonBack:hover {
	border-image: url(:/images/ui/login_back_hover.png);
}

#widgetBottom {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #d7f5fe, stop: 1.0 #ffffff);
/*	border-bottom-left-radius: 3px;
	border-bottom-right-radius: 3px;*/
	border-top:1px solid #035388;
}

#widgetBottom #labelPhoto {
	background-image: url(:/images/ui/login_photo.png);
	border-radius: 50px;
}

#widgetBottom #labelErrorMsg {
	background: #ffffbf;
	border-radius: 5px;
}

#widgetBottom QLineEdit {
	border: 1px solid #A2BDCE;
	border-radius: 3px;
	background-repeat: none;
	padding-left: 36px;
}

#widgetBottom #lineEditUserName {
	background-image: url(:/images/ui/login_username.png);
}

#widgetBottom #lineEditPassword {
	background-image: url(:/images/ui/login_password.png);
}

#widgetBottom #pushButtonForgetPassword {
	border: none;
	color: #0066ff;
	text-decoration: underline;
	text-align: right;
}

#widgetBottom #pushButtonLogin {
	color: #ffffff;
	font-size: 16px;
	border-image: url(:/images/ui/bi_login.png) 0 3 0 3;
	border-top: 0px transparent;
	border-right: 3px transparent;
	border-bottom: 0px transparent;
	border-left: 3px transparent;
	text-align: center;
}

#widgetBottom #pushButtonLogin:hover {
	border-image: url(:/images/ui/bi_login_hover.png) 0 3 0 3;
}

#widgetBottom #pushButtonLogin:pressed {
	border-image: url(:/images/ui/bi_login_pressed.png) 0 3 0 3;
}

#widgetBottom #labelLoginVersion {
	font-size: 12px;
	color: #666666;
	padding-top: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTop" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>63</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>63</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonBack">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetBottom" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widgetLoginPanel" native="true">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>21</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="ns::ElidedLabel" name="labelPhoto">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="labelErrorMsg">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>13</number>
              </property>
              <property name="topMargin">
               <number>13</number>
              </property>
              <item>
               <widget class="QLineEdit" name="lineEditUserName">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>Username/Mobile/Email</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widgetPassword" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>3</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="lineEditPassword">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                   <property name="placeholderText">
                    <string>Password</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widgetPasswordPanel" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <property name="margin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QCheckBox" name="checkBoxSavePassword">
                      <property name="text">
                       <string>Save Password</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButtonForgetPassword">
                      <property name="cursor">
                       <cursorShape>PointingHandCursor</cursorShape>
                      </property>
                      <property name="text">
                       <string>Forget Password</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonLogin">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="text">
                 <string>Login</string>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelLoginVersion">
        <property name="text">
         <string>Version : 1.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ns::ElidedLabel</class>
   <extends>QLabel</extends>
   <header>elidedlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections/>
</ui>
